api/notd/api/models_v1.py

```diff
@@ -212,3 +212,22 @@ class ApiGmCollectionRow(BaseModel):
 class ApiLatestAccountGm(BaseModel):
     accountGm: ApiAccountGm
     accountCollectionGms: List[ApiAccountCollectionGm]
+
+
+class ApiCollectionOverlap(BaseModel):
+    collectionOverlapId: int
+    createdDate: datetime.datetime
+    updatedDate: datetime.datetime
+    registryAddress: str
+    galleryAddress: str
+    ownerAddress: str
+    tokenCount: int
+    galleryCount: int
+
+
+class ApiGalleryCollectionOverlap(BaseModel):
+    galleryAddress: str
+    registryAddress: str
+    ownerCount: str
+    tokenCount: int
```

Comment:
what do these represent? i would imagine just the collectionOverlaps is necesarry and then this whole object becomes unencesarry cos we can just return the list directly?

Reply:
Owner count is the sum of all the owners that have both the registry and other_registry tokens, token Count is the total number of other_registry tokens. If i send just the overlaps, you would have to do those two calculations on your end At least that was my thought when writing this

Reply:
thats fine, we can add it up on the other end. this doesnt make sense anyway because (e.g.) whats the other registry address here, how can you pick between all the ones in the list?